lib/kitto/hooks/router.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Kitto.Hooks.Router do
+  use Plug.Router
+
+  if Mix.env == :dev, do: use Plug.Debugger, otp_app: :kitto
+  unless Mix.env == :test, do: plug Plug.Logger
+
+  plug :match
+  plug :dispatch
+
+  match "/:hook" do
```

Comment:
I thought about that. I would say no. Instead a lot of APIs when sending webhook requests will add a request signature to the HTTP headers of the request, authenticating that the webhook is coming from where it should be coming from.